Jenkinsfile.windows

```
@@ -0,0 +1,281 @@
+                            def packageName = "RStudio-${PACKAGE_VERSION}-RelWithDebInfo"
+
+                              bat "\"C:\\Program Files (x86)\\Windows Kits\\10\\bin\\10.0.19041.0\\x86\\signtool\" sign /f %pfx-file% /p %PFX_FILE% %PFX_PASS% /v /debug /n \"RStudio PBC\" /t http://timestamp.digicert.com  package\\win32\\build\\${packageName}.exe"
+                              bat "\"C:\\Program Files (x86)\\Windows Kits\\10\\bin\\10.0.19041.0\\x86\\signtool\" verify /v /pa package\\win32\\build\\${packageName}.exe"
+                          }
+                        }
+                      }
+
+                      stage('Upload') {
+                        steps {
+                          script {
+                            def buildDest = "s3://rstudio-ide-build/${current_container.flavor}/windows"
+                            def packageName = "RStudio-${PACKAGE_VERSION}"
+
+                            // strip unhelpful suffixes from filenames
+                            bat "move package\\win32\\build\\${packageName}-RelWithDebInfo.exe package\\win32\\build\\${packageName}.exe"
+                            bat "move package\\win32\\build\\${packageName}-RelWithDebInfo.zip package\\win32\\build\\${packageName}.zip"
+
+                            // windows docker container cannot reach instance-metadata endpoint. supply credentials at upload.
+                            withAWS(role: 'jenkins', roleAccount: AWS_ACCOUNT_ID) {
+                              retry(5) {
+                                bat "aws s3 cp package\\win32\\build\\${packageName}.exe ${buildDest}/${packageName}.exe"
+                                bat "aws s3 cp package\\win32\\build\\${packageName}.zip ${buildDest}/${packageName}.zip"
+                              }
+                            }
+                          }
+                        }
+                      }
+
+                      stage('Sentry Upload') {
+                        environment {
+                          SENTRY_API_KEY = credentials('ide-sentry-api-key')
+                        }
+
+                        steps {
+                          retry(5) {
+                            // convert the PDB symbols to breakpad format (PDB not supported by Sentry)
+                            timeout(activity: true, time: 15) {
+                              bat '''
+                                cd package\\win32\\build
+                                FOR /F %%G IN ('dir /s /b *.pdb') DO (..\\..\\..\\dependencies\\windows\\breakpad-tools-windows\\dump_syms %%G > %%G.sym)
+                              '''
+
+                              // upload the breakpad symbols, attempt to run sentry uplaod
+                              bat "cd package\\win32\\build\\src\\cpp && ..\\..\\..\\..\\..\\dependencies\\windows\\sentry-cli.exe --auth-token %SENTRY_API_KEY% upload-dif --log-level=debug --org rstudio --project ide-backend -t breakpad ."
```

Single quotes for secure string interpolation
```suggestion
                              bat 'cd package\\win32\\build\\src\\cpp && ..\\..\\..\\..\\..\\dependencies\\windows\\sentry-cli.exe --auth-token %SENTRY_API_KEY% upload-dif --log-level=debug --org rstudio --project ide-backend -t breakpad .'
```